src/js/components/cycleCount/CycleCount.jsx

```diff
@@ -40,6 +41,26 @@ const CycleCount = () => {
     isLoading,
   } = useCycleCountFilters();
 
+  const {
+    selectRow,
+    isChecked,
+    selectHeaderCheckbox,
+    selectedCheckboxesAmount,
+    headerCheckboxProps,
+    checkedCheckboxes,
+    resetCheckboxes,
+  } = useTableCheckboxes();
+
+  const {
+    selectRow: countSelectRow,
+    isChecked: countIsChecked,
+    selectHeaderCheckbox: countSelectHeaderCheckbox,
+    selectedCheckboxesAmount: countSelectedCheckboxesAmount,
+    headerCheckboxProps: countHeaderCheckboxProps,
+    checkedCheckboxes: countCheckedCheckboxes,
+    setCheckedCheckboxes: countSetCheckedCheckboxes,
+  } = useTableCheckboxes();
```

Comment:
Can't you call it in the toCountTab and pass the checked checkboxes to that component? By doing so, you should be able to set those values as a default. If it's not doable somehow:

Do something like:
```
const allProductsTabCheckboxes = useTableCheckboxes();
const toCountTabCheckboxes = useTableCheckboxes();
```
\+ Add comments explaining why we have to do that.
Remember that when the state that you are refreshing is in the main component (CycleCount), setting a state to a new value (setCheckedCheckboxes) will rerender the whole component, so please be sure that there are no performance issues while using huge data.